src/typescript/frontend/src/pages/emojicoin/components/trade-history/constants/index.ts

```diff
@@ -0,0 +1,56 @@
+export const HEADERS = [
+  {
+    width: "10%",
+    text: "Rank",
+  },
+  {
+    width: "11%",
+    text: "Apt",
+  },
+  {
+    width: "10%",
+    text: "🖤",
+  },
+  {
+    width: "10%",
+    text: "Type",
+  },
+  {
+    width: "10%",
+    text: "Date",
+  },
+  {
+    width: "51%",
+    text: "Transaction",
+  },
+];
+
+export const DATA = [
+  {
+    rankIcon: "🐳",
+    rank: "based",
+    apt: "1.0213",
+    emoji: "11.11M",
```

Comment:
```suggestion
    quote: "1.0213",
    base: "11.11M",
```

We'll likely refer to these as base/quote in indexer calls/type defs, you don't have to change this now since I know you're just stubbing it out but something to keep in mind